### ISSUE TITLE
Folder for relog

### ISSUE THREAD

**Issue by buhCharm** [TumblThreeApp/TumblThree#106]
Can reblog and posts be separated into a different folders? instead of everything being bundled together. Something similar to this:
Blogs--> buhCharm --->  

> Posts 
> Reblogs

![Screenshot ()](https://user-images.githubusercontent.com/75167096/100515684-cae6f380-3175-11eb-965c-519bea0493f1.png)

**Comment by thomas694** [comment]
At the moment there's no way to do that. Would it also be enough to have a marker (e.g. ...[R].jpg) in the filename? A file rename functionality is discussed and maybe it could be considered there.

Would this feature (separate folders) be helpful for others too?

**Comment by buhCharm** [comment]
Yes, i believe it will be helpful to differentiate the post of the user and what was reblogged. I just think it makes it easier than everything being stored together. Anything that can be used to idenity what was posted by the user compare to what was reblogged will be helpful it makes it more organised. Thank you for your reply and for fixing my typo

**Comment by Someonemustnothavethis** [comment]
I like this idea and would find it highly helpful.  Any method that would differentiate original content from the huge overwhelming mountain of tired old reblogs would be appreciated.

**Comment by thomas694** [comment]
In the meantime there are two options that could help you.
First, you can disable the setting "download reblogged posts" if you don't like to download them at all.
Second, if you like to have them, but would like to distinguish them, you can use the filename template with the %r token, so that every file from a reblogged post has the word "reblog" in its name.

**Comment by Someonemustnothavethis** [comment]
Thanks for the tip Thomas, that's working fine as I type.
